test/context-config.test.js

```diff
@@ -145,30 +145,29 @@ test('config without exposeHeadRoutes', t => {
     handler
   })
 
-  fastify.inject({
+  let response = await fastify.inject({
     method: 'GET',
     url: '/get'
-  }, (err, response) => {
-    t.error(err)
-    t.equal(response.statusCode, 200)
-    t.same(JSON.parse(response.payload), Object.assign({ url: '/get', method: 'GET' }, schema.config))
   })
 
-  fastify.inject({
+  t.assert.ifError(response.error)
+  t.assert.equal(response.statusCode, 200)
```

Comment:
We should use `strictEqual` here over `equal`
```suggestion
  t.assert.strictEqual(response.statusCode, 200)
```